### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#10653] [General]
Any idea when the next preview or RC version will be released?

**Comment by kyuranger** [comment]
I am looking forward to it also. But I don't know what will be fixed or updated in the next release.  It seems Avalonia team has been focusing on Avalonia XPF recently.

**Reply by kekekeks** [reply]
Note that "XPF focus" usually means adding missing features to Avalonia, since our policy is to make thin wrappers on top of Avalonia APIs when possible.  It also gives us better understanding why certain features were designed in a particular way in WPF which would allow porting stuff later (people were complaining about missing Storyboard support for years and we are now investigating how they work internally and what it would take to bring those to Avalonia).

The recent example would be the ongoing [dispatcher rework](https://github.com/AvaloniaUI/Avalonia/issues/10520) and reworking layout/render/animation ordering which would result in less wasted CPU cycles, get rid of "animation isn't properly applied on the first frame" problem and fix "input is being ignored because layout/rendering can't keep up" issues.

You are free to complain about people not doing more stuff for free though, that's completely understandable.

**Reply by kyuranger** [reply]
Thanks for replying and your hard work on Avalonia. Before your reply, I didn't know the relation between Avalonia UI and Avalonia XPF. Previously, all of us were looking forward to a new version of Avalonia UI but suddenly released a brand-new project without any advance notice. Whereas, most of us should not complain because we did not contribute any to Avalonia.

**Reply by kyuranger** [reply]
Kindly fix this if possible in RC version: https://github.com/wieslawsoltes/Svg.Skia/discussions/82
It still exists in 11.0.0-Preview5 and its Maintainer said it is bug of Avalonia.

**Reply by timunie** [reply]
@kyuranger the team will define the priorities as they have a really good overview about which topics are the most important. I cannot say if the linked issue is one of those or if there is already a fix in latest nightly builds. 
However, if you are facing a bug that is urgent *for you*, we also offer paid support where you can put priority on issues that you are facing in particular. Moreover you are welcome to file PRs (after 11.0 is done, as community PRs are paused in favor of capacity for 11.0 release) to fix issues you have.

Thank you for your understanding and happy coding
Tim

**Reply by kyuranger** [reply]
> @kyuranger the team will define the priorities as they have a really good overview about which topics are the most important. I cannot say if the linked issue is one of those or if there is already a fix in latest nightly builds. However, if you are facing a bug that is urgent _for you_, we also offer paid support where you can put priority on issues that you are facing in particular. Moreover you are welcome to file PRs (after 11.0 is done, as community PRs are paused in favor of capacity for 11.0 release) to fix issues you have.
> 
> Thank you for your understanding and happy coding Tim

Thank you so much

**Reply by mysteryx93** [reply]
That bug has been there forever and unimpactful on the framework structure, I doubt it will be considered high-priority for v11 release

**Reply by kyuranger** [reply]
> That bug has been there forever and unimpactful on the framework structure, I doubt it will be considered high-priority for v11 release

Currently I can't create an opacity animation to SVG for this bug while WPF can when I migrating WPF project to Avalonia. I feel very helpless.

**Reply by timunie** [reply]
as a workaround, can't you render a Rectangle above and animate its opacity in "reverse" direction?

**Reply by kyuranger** [reply]
> as a workaround, can't you render a Rectangle above and animate its opacity in "reverse" direction?

Thank you. Let me try this.

**Comment by kekekeks** [comment]
[Filtering by 11.0 milestone](https://github.com/AvaloniaUI/Avalonia/issues?q=is%3Aopen+is%3Aissue+milestone%3A11.0 ) would give you a more or less accurate list of remaining work we want to do before the RC. It doesn't include private customer issues, but there is only a few of them right now.

**Reply by mysteryx93** [reply]
So they're heading for a RC next? If it was for a Preview that can at least publish with trimming, that could happen soon. If they aim for a RC, we can easily look for another month. I'm kind of waiting for it to publish an app.

**Reply by kekekeks** [reply]
Preview was planned last week, but is currently blocked by DBus-related crash on Linux. Once that's resolved there will be a preview release from the current master.
